README.md

```diff
@@ -103,7 +103,19 @@ $ docker run -d \
 
 * Then start any containers to be proxied as described previously.
 
-Note: If the docker-gen container name is static and you want to explicitly set it, use `-e NGINX_DOCKER_GEN_CONTAINER=nginx-gen`. The same thing is true with the nginx container (`-e NGINX_PROXY_CONTAINER=nginx`).
+Note: 
+If the 3 containers are using using static names, as shown in the examples. The label (`--label com.github.jrcs.letsencrypt_nginx_proxy_companion.docker_gen`) on `nginx` container can be removed and additional evironment variables set on the `letsencrypt-nginx-proxy-companion` container. The variables are `-e NGINX_DOCKER_GEN_CONTAINER=nginx-gen` and `-e NGINX_PROXY_CONTAINER=nginx`.
```

Comment:
The variables themselves are just `NGINX_DOCKER_GEN_CONTAINER` and `NGINX_PROXY_CONTAINER`, not the whole thing with the -e switch and the variable value.